src/components/Tables/Table.tsx

```diff
@@ -444,8 +444,12 @@ const Table = <Data,>({
                         sortServer={sortConfig.sortPossible}
                         onSort={handleSort}
                         progressComponent={
-                            <Centerer>
-                                <CircularProgress aria-label="table-progress-bar" />
+                            <Centerer role="rowgroup">
+                                <CircularProgress
+                                    role="row"
```

Comment:
can't we use the `progressbar` role?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role

Reply:
No, the table specifically needs children with roles "row" and "rowgroup". I tried to override this requirement by setting aria-busy={isLoading} on the table component and various divs above of it in the hierarchy, which supposedly should work, but unfortunately it wouldn't. I think aria-busy needs to be set to true on the underlying table component, like, the library's component, but there's no way to do this unfortunately, think it's an oversight on their behalf. I thought this was the best fix.

Reply:
let me know your thoughts

Reply:
I don't think I can merge because the requested changes haven't been addressed